nevergrad/functions/perfcap3d/core.py

```diff
@@ -0,0 +1,436 @@
+import json
+import os
+import pathlib
+import typing as tp
+import logging
+import itertools
+import copy
+from enum import Enum
+from uuid import uuid4 as uuid_gen
+from nevergrad.parametrization import parameter as p
+from .rmqservice import RMQService as _RMQService, RMQSettings as _RMQSettings
+from ...functions.base import ExperimentFunction
+
+class Status(Enum):
+    OK = 0
+    Failed = 1
+
+class VarNormalizer:
```

Comment:
cant these constraints be integrated within the parametrization? It's better if all this is handled in one place

Reply:
I am not sure about this. Can you point me to the correct direction ?
VarNormalizer normalizes domain to -1,1 in order to cope with optimizers that may have scaling issues wrt to parameters